src/actions/workspaces.ts

```diff
@@ -46,9 +47,14 @@ export const changeExecTime = (execTime: string, workspaceLocation: WorkspaceLoc
 export const changeSideContentHeight = (height: number, workspaceLocation: WorkspaceLocation) =>
   action(actionTypes.CHANGE_SIDE_CONTENT_HEIGHT, { height, workspaceLocation });
 
-export const chapterSelect = (chapter: number, workspaceLocation: WorkspaceLocation) =>
+export const chapterSelect = (
```

Comment:
This function is used by several `Workspaces` (not just the `Playground`, but also `Sourcecast`, `Sourcereel` and `Grading`. 

Since we modified the type signature of this function, we need to modify the application of this function in these workspaces.

Reply:
For the Playground workspace, when this function is applied, the `variant` argument is obtained from the chapter menu.
For the other workspaces, when this function is applied, a `default` variant is passed as the argument